seacrowd/config_helper.py

```diff
@@ -0,0 +1,1038 @@
+"""
+Utility for filtering and loading SEACrowd datasets.
+"""
+from collections import Counter
+from importlib.machinery import SourceFileLoader
+import logging
+import os
+import pathlib
+from tqdm import tqdm
+from types import ModuleType
+from typing import Callable, Iterable, List, Optional, Dict
+
+from dataclasses import dataclass
+from dataclasses import field
+import datasets
+
+from .utils.configs import SEACrowdConfig
+from .utils.constants import Tasks, SCHEMA_TO_TASKS
+import pandas as pd
+
+_LARGE_CONFIG_NAMES = [
```

Comment:
are we going to update it later? or we just write it as-is from NusaCrowd?

Reply:
Will update later! Might also deprecate this function depending on its use case. Do you have any suggestion regarding this?

Reply:
prob we can change the value according to the SEACrowd Dataset Size Info. But since NusaCrowd v1.0 is a subset of SEACrowd, we can keep it as-is for now and update it later